tests/multi_net/sslcontext_server_client_ciphers.py

```diff
@@ -4,6 +4,7 @@
     import os
     import socket
     import ssl
+    import tls
```

Comment:
Since this test is MicroPython-specific, may as well remove the `import ssl` and change all uses of `ssl` in this test to `tls` (should be easy).